dvc/repo/experiments/save.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+from typing import TYPE_CHECKING, List, Optional
+
+from funcy import first
+
+if TYPE_CHECKING:
+    from dvc.repo import Repo
+
+
+logger = logging.getLogger(__name__)
+
+
+def save(
+    repo: "Repo",
+    name: Optional[str] = None,
+    force: bool = False,
+    include_untracked: Optional[List[str]] = None,
+) -> Optional[str]:
+    """Save the current workspace status as an experiment.
+
+    Returns the saved experiment's SHAs.
+    """
+    queue = repo.experiments.workspace_queue
+    logger.debug("Saving workspace in %s", os.getcwd())
+
+    staged, _, _ = repo.scm.status(untracked_files="no")
```

Comment:
Since we're checking for untracked files above, how about:
=
```suggestion
    staged, unstaged,untracked = repo.scm.status(untracked_files="no")
    if untracked:
        logger.warning(
            "The following untracked files were present in "
            "the workspace before saving but "
            "will not be included in the experiment commit:\n"
             "\t%s",
             ", ".join(untracked),
        )
```

Reply:
Let's make this change and remove the other call, but we need to handle `include_untracked`